jobs/harbor/templates/bin/pre-start.erb.sh

```diff
@@ -144,18 +144,22 @@ function installHarbor() {
 # Check existing Harbor Version
 checkHarborVersion() {
   if [ -z "$INSTALLED_HARBOR_VERSION" ]; then
-     if $DOCKER_CMD images | grep vmware/harbor-log ; then
-       # Harbor was installed which does not write the HARBOR_VERSION_FILE.
-       echo 1
-     else
        # Harbor was not installed on this machine before.
        echo 2
-     fi
   else
     compareVersion $HARBOR_FULL_VERSION $INSTALLED_HARBOR_VERSION
   fi
 }
 
+backupHarborDB() {
+  result=$(checkHarborVersion)
+  if [ $result -le 0 ]; then
+    return
+  fi
+  timestamp=$(date +"%Y-%m-%d-%H-%M-%S")
```

Comment:
1） I don't think we need the second in the directory name
2) how do we clean the back dir?